### DISCUSSION TITLE
Ask questions about raylib.v here!

### DISCUSSION THREAD

**Discussion by irishgreencitrus** [irishgreencitrus/raylib.v#10] [Announcements]
## 👋 Hey there!
This is a place to ask questions about the usage of the library.
Example discussion types:
- How to setup the library.
- How to do X in the library.
- Problems you're not sure how to form into an issue.
- A problem that is only relevant to a specific setup.
- Discuss implementation details
- Discuss new feature requests.

**Comment by Miniputs** [comment]
Will there be support for raylib 4.5?

**Reply by irishgreencitrus** [reply]
Yes, I'll start working on updating soon

**Comment by SCWilly** [comment]
Ia there a way to supress the raylib console on windows. Thanks.

**Comment by MatejMagat305** [comment]
Can I at summer make fokus to adapt raylib.v to raylib 5.x and make fokus to android according https://github.com/gen2brain/raylib-go?
